### PR TITLE
WIP: Allow openjdk test of jdknext

### PR DESCRIPTION
**Pull request by andrew-m-leonard**
Signed-off-by: andrew-m-leonard <andrew_m_leonard@uk.ibm.com>

### CODE REVIEW

File: openjdk/build.xml [hunk 1]
```diff
@@ -82,8 +95,8 @@
 		<if>
 			<contains string="${JVM_VERSION}" substring="openj9"/>
 			<then>
-				<property name="jdkName" value="openj9-openjdk-jdk${JDK_VERSION}" />
-				<property name="regressionRepo" value="https://github.com/ibmruntimes/openj9-openjdk-jdk${JDK_VERSION}.git" />
+				<property name="jdkName" value="${openjdkOpenJ9Git}" />
+				<property name="regressionRepo" value="https://github.com/andrew-m-leonard/${openjdkOpenJ9Git}.git" />
```
Comment (smlambert) [inline openjdk/build.xml]: Still pointing to your personal fork

File: openjdk/build.xml [hunk 2]
```diff
@@ -92,7 +105,7 @@
 		</if>
 		<echo message="git clone --depth 1 -q ${gitReleaseTag} ${regressionRepo}" />
 		<exec executable="git" failonerror="false">
-			<arg line="clone --depth 1 -q ${gitReleaseTag} ${regressionRepo}" />
+			<arg line="clone --depth 1 -b jdilistener ${regressionRepo}" />
```
Comment (smlambert) [inline openjdk/build.xml]: Still pointing to your branch

**Comment by andrew-m-leonard** [comment]
@ShelleyLambert  
sorry this is for my onwn purpose at the moment....
I will raise a new issue for this as it needs some thought...

**Comment by smlambert** [comment]
no worries, @andrew-m-leonard - let us know if you need help ...

**Comment by smlambert** [comment]
related: https://github.com/AdoptOpenJDK/openjdk-tests/pull/1110

**Pull request closed without merging**